commands/economy/rob.js

```
@@ -0,0 +1,199 @@
+            case "cooldown":
+                const embed = new EmbedBuilder()
+                    .setColor(ECONOMY_EMBED_COLOR)
+                    .setTitle("Rob Cooldown")
+                    .setDescription("You need to wait before attempting another robbery!")
+                    .addFields({
+                        name: "Next rob available",
+                        value: formatRelativeTimestamp(result.nextRobAt),
+                        inline: true,
+                    })
+                    .setFooter({ text: `Guild: ${interaction.guild?.name || "Unknown"}` })
+                    .setTimestamp();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)